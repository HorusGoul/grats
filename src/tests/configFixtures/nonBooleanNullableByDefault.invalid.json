{
  "grats": { "nullableByDefault": "NO!" },
  "include": ["../test.ts"]
}
