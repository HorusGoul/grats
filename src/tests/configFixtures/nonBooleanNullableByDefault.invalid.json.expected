-----------------
INPUT
----------------- 
{
  "grats": { "nullableByDefault": "NO!" },
  "include": ["../test.ts"]
}

-----------------
OUTPUT
-----------------
src/tests/configFixtures/nonBooleanNullableByDefault.invalid.json:2:35 - error: Expected `nullableByDefault` to be a boolean if provided.

2   "grats": { "nullableByDefault": "NO!" },
                                    ~~~~~
